src/pgtest/src/lib.rs

```diff
@@ -94,7 +120,7 @@ impl PgTest {
             Message::AuthenticationOk => {}
             _ => bail!("expected AuthenticationOk"),
         };
-        pgtest.until(vec!["ReadyForQuery"], vec![], HashSet::new())?;
+        pgtest.until(vec!["ReadyForQuery"], vec!['C', 'S', 'M'], HashSet::new())?;
```

Comment:
```suggestion
        pgtest.until(vec!["ReadyForQuery"], vec!['C', 'M', 'S'], HashSet::new())?;
```

pure nitpick change to keep all the vecs in the same order.